node-test/src/main/scala/org/bitcoins/node/Bloom.scala

```diff
@@ -0,0 +1,90 @@
+package org.bitcoins.node
```

Comment:
This file can be disregarded until the PR is merge-ready, this is a nice main method to have available when developing